rollup.config.js

```diff
@@ -0,0 +1,53 @@
+import vue from 'rollup-plugin-vue'
+import replace from "@rollup/plugin-replace";
+import commonjs from '@rollup/plugin-commonjs';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+
+import pkg from './package.json';
+
+const commonPlugins = [
+  replace({
+    preventAssignment: true,
+    values: {
+      'import.meta.env.VERSION': JSON.stringify(pkg.version)
+    }
+  }),
+]
+
+export default [
```

Comment:
You forgot `external` option, to not include dependencies in bundle. It requiers `nodeResolve` plugin.